lib/browser/NoopBrowserRunner.js

```diff
@@ -0,0 +1,38 @@
+function NoopBrowserRunner() {
+}
+
+NoopBrowserRunner.prototype._noop = function () {
+    return new Promise(function (fulfill) {
+        fulfill();
+    });
+};
+
+NoopBrowserRunner.prototype.terminate = NoopBrowserRunner.prototype._noop;
+
+NoopBrowserRunner.prototype.init = NoopBrowserRunner.prototype._noop;
+
+NoopBrowserRunner.prototype.loadOrigin = NoopBrowserRunner.prototype._noop;
+
+NoopBrowserRunner.prototype.run = function (fn) {
+    return new Promise(function (fulfill, reject) {
+        try {
+            fn(fulfill);
+        } catch (e) {
+            reject(e);
+        }
+    });
+};
+
+NoopBrowserRunner.prototype.getExceptions = function () {
+    return [];
+};
+
+NoopBrowserRunner.prototype.proxy = function (fn, done) {
+    fn(done);
+};
+
+NoopBrowserRunner.prototype.getOrigin = function () {
+    return null;
+};
+
+module.exports = NoopBrowserRunner;
```

Comment:
All these now have a new line, not sure why it isn't removing your comment, see https://github.com/dpwspoon/k3po.js/commit/7ed7dae50f45d059669fb5bd60fa5dc04b537c8b for details